manifests/host.pp

```diff
@@ -11,7 +11,7 @@
 
   $host = $name
 
-  include dhcp::params
+  include ::dhcp::params
```

Comment:
@bastelfreak Any reason behind not inheriting the params class here like you did for failover.pp and disable.pp?